initScripts/x86_64__Ubuntu_16.04__Docker_17.06.sh

```diff
@@ -235,6 +237,21 @@ setup_mounts() {
 }
 
 setup_envs() {
+  local docker_client_location=$LEGACY_CI_DOCKER_CLIENT_LEGACY
+  local is_docker_legacy=true
+
+  local installed_docker_version=$DOCKER_VERSION
+  if [ -z "$installed_docker_version" ]; then
+    installed_docker_version=$(sudo docker version --format {{.Server.Version}})
+  fi
+
+  if [ -f "$LEGACY_CI_DOCKER_CLIENT_LATEST" ]; then
+    is_docker_legacy=false
+    docker_client_location=$LEGACY_CI_DOCKER_CLIENT_LATEST
+  fi
+
+  __process_msg "Docker client location on host: $docker_client_location"
```

Comment:
this script uses `echo` for printing most stuff. `process_msg` is used here and in `pull_cexec`. Should I be using `echo` in these places to maintain uniformity ?

Reply:
ya. lets change it to echo for now. we'll refactor this to use process_msg globally before releasing